src/interfaces/package-export-transport.ts

```diff
@@ -48,18 +48,16 @@ export interface PackageKeyAndVersionPair {
 export interface NodeExportTransport {
     key: string;
     parentNodeKey: string;
-    packageNodeKey: string;
```

Comment:
Why did we remove the package node key?

Reply:
The property is not used in CLI, and having it defined or not doesn't impact the parse/stringify steps during import.